unload_databricks_data_to_s3.py

```diff
@@ -102,7 +101,13 @@ def export_meta_data(event_count: int, partition_count: int):
     parser.add_argument("--s3_region", nargs='?', default=None, help="s3 region")
     parser.add_argument("--s3_endpoint", nargs='?', default=None, help="s3 endpoint")
     parser.add_argument("--s3_path", required=True, help="s3 path where data will be written into")
-    parser.add_argument("--max_records_per_file", help="max records per output file", nargs='?', type=int, default=MAX_RECORDS_PER_OUTPUT_FILE, const=MAX_RECORDS_PER_OUTPUT_FILE)
+    parser.add_argument("--max_records_per_file", help="max records per output file", nargs='?', type=int,
+                        default=MAX_RECORDS_PER_OUTPUT_FILE, const=MAX_RECORDS_PER_OUTPUT_FILE)
+    parser.add_argument("--disable_append_only_filter",
```

Comment:
In order to support mutation ingestion, felt like we need 2 config actually that 1 for specifying which mutation actions to be supported and 1 for specifying which cargo pipeline(mutation vs regular one). The first config will decide what data will be imported, and 2nd config will decide how data will be formatted before saving to S3. For regular databricks import, now it can ingest both `update` && `insert` changes so it also needs config for specifying mutation actions.

Felt like having 2 configs serves our long term goals better if we want to allow customer decide which actions to be imported. Then maybe we should do that at very beginning to avoid the big pain in changing arguments in future.

Reply:
this is the responsibility of the code that will live in Falcon. Please, wait for the Falcon PR to understand the context

Reply:
also, in long-term, `build_sql_to_query_table_of_version` and `build_sql_to_query_table_between_versions` should be moved out from this script to Falcon repo and `filter_data` should be incorporated into the SQL queries generated by `build_sql_to_query_table_between_versions` to improve filtering performance

Reply:
Updated the PR description with a reference to Falcon PR, which handles the main mutation sql generation logic

That sql generation should also handle flows represented in `build_sql_to_query_table_of_version`, `build_sql_to_query_table_between_versions`, and `filter_data`

Reply:
ok, if we want to skip filtering for mutation ingestion, let's change rename the flag to something else like `--is_mutation_ingestion` or similar. `--disable_append_only_filter` is a confusing naming since regular ingestion supports both `insert` && `update` so ideally it can be `false` for regular ingestion which is actually not true.